snippets/tf/cache/cache.tf

```diff
@@ -0,0 +1,34 @@
+variable "region" {}
+
+variable "region_short" {}
+
+variable "cache_node_size" {}
+
+variable "cache_port" {}
+
+variable "cache_num_nodes" {}
+
+variable "cache_param_group" {}
+
+variable "cache_engine_version" {}
+
+variable "cache_subnet_ids" {}
+
+variable "cache_security_group" {}
+
+resource "aws_elasticache_subnet_group" "shared-redis-subnet-group" {
+  name       = "shared-redis-subnet-group"
+  subnet_ids = ["${split(",",var.cache_subnet_ids)}"]
```

Comment:
I know this line cost you some time figuring this out, maybe a comment with a link to the tf issue that makes this workaround necessary would be helpful for future reference?